spec/routes/api/cli/golden-files/--help.txt

```diff
@@ -1,8 +1,17 @@
-Usage: ubi [options] [subcommand [subcommand-options] ...]
+CLI to interact with Ubicloud
+
+Usage:
+    ubi [options] [command [command-options] ...]
+
+Commands:
```

Comment:
It might be helpful to include a hint to the effect of: `run `ubi help [command]` for more info` in here somewhere.

Reply:
OK.  I'll work on adding that (requires changes to rodish).